### ISSUE TITLE
French language

### ISSUE THREAD

**Issue by Elytherion** [Elytherion/tdeheroes#69]
Is there anyone who backed the French kickstarter and would like to provide the French translations once the books have been shipped? I am not able to speak French at all, but I would like to provide a French version of my app.

**Comment by br00n0** [comment]
Guten Tag, I'm working on the Scriptarium team for the upcoming French Oeil Noir version. We will be able to help you out for a French version of your application. Please contact me to prepare a plan! Thanks!

**Comment by dyvim** [comment]
@Elytherion @br00n0 Hello. any news about this translation? Have you been in touch?

**Comment by Elytherion** [comment]
Yeah, we talked about a possible localization for the French community and ran into the following issues (they did not came up during our conversation only but also when I was thinking about how the work should be done in general):

- **Licensing:** Black Book Editions have the French license, but they want to publish the app on their website for free and do not want to have their translation in the Scriptorium, whereas Ulisses requires the Scriptorium and does not want their English translation/original German version to appear outside the Scriptorium. Basically, I'd have to split the app. But, although I definitely respect the decisions of the publishers, it feels kinda strange to split the apps (both of them you could download for free) although you are actually one community. So, what I'm currently checking is how much time I would need to build the mobile apps. At least the iOS-App would require the App Store and thus could not be published to the Scriptorium at all, which would require me to talk to Ulisses about an exemption from the Scriptorium rules for the app. And I also would have to talk to @br00n0 if a mobile app is okay for them, too. In this case I'd probably be able to have all languages in one mobile app and one desktop app, which would be best.
Just in case: This is not meant as an offense or something similar, I just want to share what's the current state and what I'm planning so that everyone is happy with the result.
- **Work on actual translation:** Due to the fact that I originally didn't expect that much interest in using the app and helping to develop the app further, I did not prepare my files for that. That's why I'm still thinking of if the current way my helpers and me are dealing with the raw data from the books is the best. I reworked so parts of them recently but I had no time to actually try it out. I've also been a little short in time, which is why things are a bit slower in general &ndash; which is certainly not what I am aiming at. I definitely don't want them to jump in at the deep end! ;) I thinks it's better to take more time to get a better result or a better way of dealing with the work.

I hope this helps to know where we are at and why I think we're still a fair way off from a full translation.

BR

**Comment by Elytherion** [comment]
I mean, @br00n0 already said they would do the translation – which would be so cool!!!! But – as you might have noticed – I‘m not a big fan of splitting the app. It would increase the work to publish a new version and – which is way more important – it would not be possible for French players to use German or English (or Dutch, which I will also put in the app in the Scriptorium) and the same in the other direction – or at least it would be no fun at all. That’s why I’m trying to keep everything for the users as simple as possible.

**Comment by dyvim** [comment]
Thanks for the explanations. I hope this licensing problem will be resolved

**Comment by Elytherion** [comment]
Update: BBE allowed me to publish the French version of Optolith to the Scriptorium! Really, that is so nice of them; I'm so happy! Bruno is going to insert all the translation, so, I guess, there will be a French v1.1 or v1.2 as well! 🎉

**Comment by dyvim** [comment]
Cool. That's a great news!
Thanks for the information.

BRgds

Le mer. 17 avr. 2019 20:40, Lukas Obermann <notifications@github.com> a
écrit :
